posthog/models/action/action_step.py

```diff
@@ -51,6 +51,7 @@ class ActionStep(models.Model):
 @receiver(post_save, sender=ActionStep)
 def action_step_saved(sender, instance: ActionStep, created, **kwargs):
     instance.action.refresh_bytecode()
+    instance.action.save()
```

Comment:
What's this for? `refresh_bytecode` should already save when needed

Reply:
Not anymore. I changed it so it doesn't save. More of an aside but models normally shouldn't save themselves - not that we really practice DDD or anything but you know :D